index.html

```diff
@@ -0,0 +1,130 @@
+                    with experts.
+                    </p>
+                    <h2 class="conf-date">2022.07.30(Sat) ~ 31(Sun)</h2>
+                    <p class="location-text">@ PrideInn Beach Resort, Mombasa, Kenya</p>
+                </div>
+            </section>
+        </main>
+        <section class="mainProgram">
+            <h2 class="main-p-heading">Main Program</h2>
+            <hr class="line-main-p">
+            <div class="programList">
+                <ul class="main-p-container">
+                    <li><img src="./images/imageIconType/LectureIcon.png" alt=""></li>
+                    <li><h2 class="sub-header">Lecture</h2></li>
+                    <li><p class="sub-text">We take a systemic approach 
+                        to help software development organizations become better at what they do 
+                        and more fun places to work.</p></li>
+                </ul>
+                <ul class="main-p-container">
+                    <li><img src="./images/imageIconType/ExhbIcon.png" alt=""></li>
+                    <li><h2 class="sub-header">CC Exhibition</h2></li>
+                    <li><p class="sub-text">Appreciate how spotify increase productivity and Microverse has unique way to learn
+                        remote pair programming and how it can benefit you and your organization.</p></li>
+                </ul>
+                <ul class="main-p-container">
+                    <li><img src="./images/imageIconType/forumIcon.png" alt=""></li>
+                    <li><h2 class="sub-header">Forum</h2></li>
+                    <li><p class="sub-text">This great time to share your thoughts
+                        and opinions with experts for each topic and summit comments/questions.</p></li>
+                </ul>
+                <ul class="main-p-container">
+                    <li><img src="./images/imageIconType/workshopIcon.png" alt=""></li>
+                    <li><h2 class="sub-header">Workshop</h2></li>
+                    <li><p class="sub-text">Try to collaborate your own work using VS code
+                        rather than just watching. How fun pairing programming can be.</p></li>
+                </ul>
+                <ul class="main-p-container">
+                    <li><img src="./images/imageIconType/igniteIco.png" alt=""></li>
+                    <li><h2 class="sub-header">CC Ignite</h2></li>
+                    <li><p class="sub-text">Get opportunities to network with cc affiliates
+                        around the world, also after the summit.</p></li>
+                </ul>
+            </div>
+        </section>
+        <div class="join-program">
+            <h2 class="join-mobile">Join CC Global Collaborative Software Development 2022</h2>
+            <h2 class="join-desktop">See the whole program</h2>
+        </div>
+        <section id="speakers">
+            <h2 class="speakers">Feature Speaker</h2>
```

Comment:
- [ Optional ] Please since you have already created a button in the speaker section why you don't try to make it work if i.e the  user can hide and show some of the speakers
![image](https://user-images.githubusercontent.com/74744787/181387975-a071368f-8223-438a-87ee-1b8ab02a7034.png)

you can add a class called to hide in your CSS like this 
.hide {
display:none;
} 
and if the user clicks the button you just have to add the class to the class list of the speakers you want to hide .